skimage/filters/_median.py

```diff
@@ -68,6 +71,12 @@ def median(image, footprint=None, out=None, mode='nearest', cval=0.0,
     >>> med = median(img, disk(5))
 
     """
+    if footprint is not None and isinstance(footprint, int):
```

Comment:
Since
```python
>>> isinstance(None, int)
False
```
This is equivalent to
```suggestion
    if isinstance(footprint, int):
```